plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPageForFlexProjectTest.java

```diff
@@ -16,28 +16,22 @@
 
 package com.google.cloud.tools.eclipse.appengine.deploy.ui;
 
-import com.google.cloud.tools.eclipse.test.util.project.TestProjectCreator;
-import java.util.Arrays;
 import org.eclipse.core.resources.IProject;
-import org.eclipse.jst.common.project.facet.core.JavaFacet;
-import org.eclipse.jst.j2ee.web.project.facet.WebFacetUtils;
 import org.junit.Ignore;
-import org.junit.Rule;
 
-// until Flex facet is enabled
+// TODO: uncomment when Flex facet is enabled
```

Comment:
Couldn't think of a good way to use `assume...()`, because the situation here is that you cannot even instantiate a test class.

I ended up adding the `.flex` bundle to `Required-Bundle` in this test bundle.